src/po/Make_mvc.mak

```diff
@@ -524,11 +556,13 @@ update-po: $(MOFILES:.mo=)
 
 # Don't add a dependency here, we only want to update the .po files manually.
 $(LANGUAGES):
-	@$(MAKE) -nologo -f Make_mvc.mak GETTEXT_PATH="$(GETTEXT_PATH)" $(PACKAGE).pot
```

Comment:
why are you removing `-nologo`

Reply:
No, no, I didn't delete "-nologo". The "-l" switch is a synonym for "-nologo". It's just not used very often and not documented much anywhere. This way the string is shorter and fits into 80 columns.